src/terminal.cpp

```diff
@@ -2813,7 +2813,10 @@ void Terminal::consumeCSI()
     // ESC [ f : HVP, Move cursor to the indicated row, column
     case 'H':
     case 'f':
-      setCursorPos(params[1], getAbsoluteRow(params[0]));
+      if (paramsCount>=2)
+       setCursorPos(params[1], getAbsoluteRow(params[0]));
```

Comment:
[minor] Wrong indentation.

Reply:
Also, please add spaces around '>='. Same for the 'FABGLIB_MAX_CSI_PARAMS-1'.

Reply:
This code has been restored to the original state, I'll rely on the clamp to do the right thing.